src/features/Admin/Users/Users.tsx

```diff
@@ -1,9 +1,102 @@
-import React from 'react'
+import axios from "axios";
+import { useContext, useEffect, useState } from "react";
+import noData from "../../../assets/images/no-data.png";
+import { AuthContext } from "../../../context/AuthContext";
+import { usersUrl } from "../../../services/api.tsx";
+import {
+  Box,
+  Container,
+  Grid,
+  Paper,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+  Typography,
+} from "@mui/material";
+
+const Users: React.FC = () => {
+  const { requestHeaders }: any = useContext(AuthContext);
+
+  const [users, setUsers] = useState([]);
+
+  // **********get all users*****************
+  const getUsersList = () => {
+    axios
+      .get(`${usersUrl}?page=1&size=10`, {
+        headers: requestHeaders,
+      })
+      .then((response) => {
+        // console.log("succ list", response?.data?.data.users);
+
+        setUsers(response?.data?.data.users);
+      })
+      .catch((error) => {
+        console.log("error", error);
+      });
+  };
+  // ************************
+  useEffect(() => {
+    getUsersList();
+  }, []);
 
-export default function Users() {
   return (
-    <div>
-      admin users
-    </div>
-  )
-}
+    <>
+      <Container>
+        <Grid item>
+          <Typography component="h2" variant="h5">
+            Users
+          </Typography>
+          <TableContainer component={Paper}>
+            <Table>
+              <TableHead sx={{ backgroundColor: '#f8f9fb' }}>
+                <TableRow>
+                  <TableCell>User name</TableCell>
+                  <TableCell>Role</TableCell>
+                  <TableCell>Phone number</TableCell>
+                  <TableCell>Email</TableCell>
+                  <TableCell>Country</TableCell>
+                </TableRow>
+              </TableHead>
+              <TableBody>
+                {users?.length > 0 ? (
+                  users.map((user) => (
+                    <>
+                      <TableRow key={user?._id}>
```

Comment:
The key prop should be given to the first child which is the fragment here <>, you can use the Fragment component to be able to add the key, or you can remove the fragment as we don't need it in the current case.
![image](https://github.com/SAMIYAghb/hotel-system/assets/64327685/d2805ce1-5d6f-48b6-bd74-2ecb12f2bf08)

Reply:
You can also destructure the user object for better clarity and extract what you need.
```
{users.map(({ userName, role, phoneNumber, email, country}) => ( ...
```
instead of needing to access the object in every table cell.